src/discord/handlers/MessageHandler.js

```diff
@@ -21,9 +21,20 @@ class MessageHandler {
     this.discord.broadcastMessage({
       username: message.member.displayName,
       message: this.stripDiscordContent(message.content),
+      replyingTo: await this.fetchReply(message),
     })
   }
 
+  async fetchReply(message) {
+    if (!message.reference) return null
+
+    const reference = await message.channel.messages.fetch(message.reference.messageID).catch(err => {
+      return null
```

Comment:
You can't use `.catch()` on a promise when you're also using `await`, you should instead wrap the statement in a `try-catch` block, and just return `null` in case an exception is thrown since we can assume the message wasn't found in that case.